docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
-    develop your team skills.
+    We'd like your help! If you're interested in improving your documentation, help us. We'll provide a resource with different experiences, and potentially
+    writers who can help with your documentation.
 
-    If you have any questions, please do not hesitate to contact me.
+    If you have any questions, please do not hesitate to contact us.
 
     Regards,
     {{ name }}
 
-Can't Find a Place to Meet?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Alternative Meetup Locations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you can't find a company to host your meetup, try something different: `meet in a cafe <http://www.meetup.com/Write-The-Docs-Seattle/events/231890982/>`_ to work on a common project, organize a picnic or do an "after-work" in a bar. Networking is an important part of meetups and it's better to meet every month than to skip one because you didn't find a sponsor.
+Don't limit your search to corporate locations. Several Write The Docs Meetups
+have successfully met in cafes, libraries, and restaurants. Some may want to
+help in exchange for the food that your members will purchase.
 
-If you're having difficulty finding a location, ask on the `Slack <http://slack.writethedocs.org>`_: you have an international community ready to help.
 
 Find attendees and speakers
 ---------------------------
 
 Attendees:
 ~~~~~~~~~~
 
-Once your meetup page is up, start promoting it. To do that:
-
-* Announce your meetup on `Write the Docs Slack <https://writethedocs.slack.com>`_ and join meetup-organizers and meetup-general channels. You can also create a dedicated channel to your meetup.
-* Announce your meetup on `Write the Docs Forum <http://forum.writethedocs.org/c/meetups>`_.
-* Create a Twitter account and connect it to your meetup's page (`example <https://twitter.com/WriteTheDocsPDX/>`_). Use it to announce your next events, live tweet your meetup, post pictures, reach new people, retweet other Write the Docs meetups, etc.
-* Use your network: ask your colleagues, friends, and contacts to promote your meetup.
-* Reach new people: find other community leaders and ask them to spread the word.
-* Attend other meetups in your area and network.
-* Add your meetup to local tech events calendars.
+Once you've set up your Meetup, promote it!
 
-No-show:
-^^^^^^^^
+* Use your network: ask colleagues, friends, and contacts to promote the meetup.
+* Attend other meetups. Tell them about Write The Docs.
+* Add your meetup to local tech event calendars.
+* Use social media. Several Write The Docs meetups publicize through Twitter (`example <https://twitter.com/WriteTheDocsPDX/>`_).
+* Help your fellow Meetup leaders. Retweet their announcements.
+* Announce your meetup on `Write the Docs Slack <https://writethedocs.slack.com>`_.
 
-A lot of people RSVPed but didn't show? Don't worry, that's one of the downsides of free meetups. Do a head-count at each event to know how many people usually attend your meetup.
 
```

Comment:
Do a head-count at each event so that you know how many people to expect, and so that you can work on improving the average.